Pods can't be terminated after kubearmor process restarts given hostpolicy is active

Kubearmor hostpolicy seems to block runc to send signals causing pods to get stuck in `Terminating` state. 

This happens only when kubearmor restarts. Tested on docker version 18.09.
Steps to reproduce this issue:
1. Run kubearmor with hostpolicy enabled.
```
vagrant@kubearmor-dev:~/KubeArmor/KubeArmor$ make && make run-with-host
```
2. Apply some hostpolicy.
```
vagrant@kubearmor-dev:~$ kubectl apply -f KubeArmor/examples/host-security-policies/hsp-kubearmor-dev-proc-path-block.yaml 
```
3. Deploy multiubuntu pods.
4. Restart kubearmor.
```
^C2021-08-12 07:06:57.244293	INFO	Got a signal to terminate the KubeArmor
2021-08-12 07:06:57.301034	INFO	Unegistered an AppArmor host profile in kubearmor-dev
2021-08-12 07:06:57.301134	INFO	Destroyed AppArmor Enforcer
2021-08-12 07:06:57.301151	INFO	Stopped the runtime enforcer
2021-08-12 07:06:59.315171	INFO	Stopped the system monitor
2021-08-12 07:06:59.315334	INFO	Terminated the KubeArmor
2021-08-12 07:07:01.316717	INFO	Terminated the gRPC service
2021-08-12 07:07:01.316782	INFO	Stopped the log feeder
2021-08-12 07:07:01.316796	INFO	Waiting for remaining routine terminations
Makefile:37: recipe for target 'run-with-host' failed
make: *** [run-with-host] Interrupt

vagrant@kubearmor-dev:~/KubeArmor/KubeArmor$ make && make run-with-host
```
5. Try to delete multiubuntu pods once kubearmor is initialized.
```
vagrant@kubearmor-dev:~$ kubectl delete -f KubeArmor/examples/multiubuntu
namespace "multiubuntu" deleted
deployment.apps "ubuntu-1-deployment" deleted
deployment.apps "ubuntu-2-deployment" deleted
deployment.apps "ubuntu-3-deployment" deleted
deployment.apps "ubuntu-4-deployment" deleted
deployment.apps "ubuntu-5-deployment" deleted
```
The logs in `/var/log/audit/audit.log` corresponding to this:
```
type=AVC msg=audit(1628750955.803:11374): apparmor="DENIED" operation="signal" profile="docker-default" pid=32187 comm="runc" requested_mask="receive" denied_mask="receive" signal=kill peer="kubearmor.host"
type=AVC msg=audit(1628750957.787:11375): apparmor="DENIED" operation="signal" profile="kubearmor.host" pid=32194 comm="runc" requested_mask="send" denied_mask="send" signal=kill peer="docker-default"
type=AVC msg=audit(1628750957.787:11375): apparmor="DENIED" operation="signal" profile="docker-default" pid=32194 comm="runc" requested_mask="receive" denied_mask="receive" signal=kill peer="kubearmor.host"
```
Status of pods:
```
vagrant@kubearmor-dev:~$ kubectl get po -n multiubuntu 
NAME                                   READY   STATUS        RESTARTS   AGE
ubuntu-1-deployment-5d6b975744-4pbbm   1/1     Terminating   0          5m28s
ubuntu-2-deployment-bc58cf44-6m76k     1/1     Terminating   0          5m28s
ubuntu-3-deployment-59bf5f5564-f9bk5   1/1     Terminating   0          5m27s
ubuntu-4-deployment-666b6dd9-prlbh     1/1     Terminating   0          5m26s
ubuntu-5-deployment-77f48cfcc7-cdl82   1/1     Terminating   0          5m25s
```

--- a/KubeArmor/enforcer/appArmorEnforcer.go
+++ b/KubeArmor/enforcer/appArmorEnforcer.go
@@ -381,12 +381,6 @@ func (ae *AppArmorEnforcer) UnregisterAppArmorHostProfile() bool {
 		return false
 	}
 
-	if err := kl.RunCommandAndWaitWithErr("apparmor_parser", []string{"-R", "/etc/apparmor.d/kubearmor.host"}); err == nil {
-		ae.Logger.Printf("Unegistered an AppArmor host profile in %s", ae.HostName)
-	} else {
-		ae.Logger.Printf("Failed to unregister the AppArmor host profile (%s)", err.Error())
-	}
-
 	if err := ae.RemoveAppArmorHostProfile(); err != nil {
 		ae.Logger.Errf("Failed to remove the AppArmor host profile in %s (%s)", ae.HostName, err.Error())
 		return false

